src/components/about-us/constants/section-02/member-img-src.js

```diff
@@ -22,10 +24,12 @@ export const membersImg = {
   EditSzuHuaChen: 'Edit-SzuHuaChen.png',
   EditTzewuChang: 'Edit-TzewuChang.png',
   EditUyo: 'Edit-Uyo.png',
-  EditYuChuanYeh: 'Edit-YuChuanYeh.png',
+  EditWenTingYen: 'Edit-WenTing.png',
+  EditFuNienTsao: 'Edit-FuNienTsao.png',
+  EditPiChi: 'Edit-Pichi.png',
   PhotoChihWeiYu: 'Photo-ChihWeiYu.png',
   PhotoJamesonWu: 'Photo-JamesonWu.png',
-  PhotoYuEnLin: 'Photo-YuEnLin.png',
+  PhotoWeiMinSu: 'Photo-WeiMingSu.png',
```

Comment:
Could we use camelCase here? such as `photoWeiMinSu`?

Reply:
Updated.